api-docs/swagger-v1.1.yml

```diff
@@ -1096,6 +1106,12 @@ parameters:
     description: restrict by health score
     enum: ['A', 'B1', 'B2', 'C1', 'C2']
     required: false
+  personal_health_scores:
+    name: personal_health_scores[]
+    type: string
+    in: query
+    description: restrict by health score, repeat parameter
+    required: false
```

Comment:
I would put in the enum here. We may need to use a different way to accept array parameters (e.g. like [here](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#path-item-object-example)), need to experiment with that if the client supports it.